Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use a Python image with uv pre-installed
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim AS uv
+
+# Install the project into `/app`
```

Comment:
I would prefer we remove comments from the Dockerfile that do not explain "why" and instead redundantly explain "what" - this comment being a great example.

However the comment on line 10 is okay with me, because it explains why `UV_LINK_MODE=copy`.

Does this compromise check out for you?